test/index.js

```diff
@@ -318,17 +318,17 @@ test('signedTypeData', (t) => {
   const sig = sigUtil.signTypedData(privateKey, { data: typedData })
 
   t.equal(utils.encodeType('Mail', typedData.types),
-    'Mail(Person from,Person to,string contents)Person(string name,address wallet)')
+    'Mail(Person from,Person[] to,string contents)Person(string name,address wallet)')
```

Comment:
I would recommend adding an independent test for this, to avoid changing the test values provided in the EIP, this way it is also ensured that the previous behaviour did not break.

Reply:
Hi @rmeissner, thanks for the feedback. I submitted an upstream PR to update the EIP test values (see ethereum/EIPs#1242.) While we don't plan to land this PR until that upstream PR also lands, you're right that separate tests would be more robust. Will update.